packages/editor/src/components/editors/event/intermediate.tsx

```diff
@@ -16,11 +16,11 @@ import {
 
 const TaskSwitchEventEditor = memo(() => {
   const name = useGeneralPart();
-  const output = useOutputPart();
   const task = useTaskPart();
   const casePart = useCasePart();
   const endPage = useEndPagePart();
-  return <InscriptionEditor icon={IvyIcons.Task} parts={[name, output, task, casePart, endPage]} />;
+  const output = useOutputPart();
+  return <InscriptionEditor icon={IvyIcons.Task} parts={[name, task, casePart, endPage, output]} />;
```

Comment:
This was not defined right? That we move the output tab on this element?
Because I don't know if this is valid. @ivy-rew what do you think here?
This order has a meaning. It defines what is done on an element one after another: https://developer.axonivy.com/doc/11.3/designer-guide/process-modeling/process-elements/index.html

> The inscription view is divided into multiple sections and the order of the sections indicates the sequence of processing. For example in the figure below, the Output mapping is performed before the code in the third section is executed.

By the way when I see this screenshot and the text above. We should also check/update the doc

Reply:
Discussed at the desk with @ivy-lli & @ivy-rew: We move the output tab to the end so that it always appears at the end (for all elements).